index.html

```diff
@@ -78,17 +78,199 @@ <h1>I’m Okeke Okechukwu.
-                    <path d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z" fill="#505F79"/>
-                  </svg>-->
-            
         </section>
+            <section class="card">
+                <div class="snap">
+                    <img src="asset/images/snapshot-portfolio.png" alt="snapshot">
+                </div>
+                <div class="wrapper">
+                <div class="details">
+                <h2>Tonic</h2>
+                <div class="canopy">
+                <span class="item1">CANOPY</span><svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8" fill="none">
+                    <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                  </svg><span class="item2">Back End Dev</span><svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" viewBox="0 0 8 8" fill="none">
+                    <circle cx="4" cy="4" r="4" fill="#C1C7D0"/>
+                  </svg><span class="item3">2015</span>
+                </div>
+                </div>
+                <div class="two">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                <div class="three">
+                    <a href="/">HTML</a><a href="/">CSS</a><a href="/">Javascript</a>
+                </div>
+                <div class="four">
+                    <a href="/">See project</a>
```

Comment:
- [ ] On hover of the button, it underlines the button, I will encourage you to remove it and add the button color interaction instead.
- [ ] I will encourage you to make the see project button have a defined width and height.
![image](https://github.com/okayboy12/Portfolio-setup-and-mobile-first/assets/86200356/3ccb9d1c-18b3-455f-ba3a-b6a7a0ffc4b4)